components/brave_news/browser/resources/shared/useFeedV2.ts

```diff
@@ -76,7 +76,7 @@ const maybeLoadFeed = (view?: FeedView) => {
   }
 
   // Don't load errored feeds.
-  if (feed.error !== undefined) {
+  if (feed.error ?? false) {
```

Comment:
I'm a little bit confused with this change. Could you tell me what's difference between the new line and the old line and with `if (feed.error)` ?

Reply:
Ah yeah, so previously I was checking only `undefined` but mojo actually sets empty undefined objects to null, so this wasn't working.

Unfortunately I can't just check for falseyness, because one of the mojom values is `0` (which is falsey), so I'm using nullish propagation (`undefined` or `null`). Thinking about it, this is still going to fail or zero.

I guess a better check is probably going to be `typeof feed.errror === 'number'` :smile: Nice catch!